shell/src/main/java/org/apache/accumulo/shell/commands/ConfigCommand.java

```diff
@@ -155,30 +156,56 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
     } else {
       // display properties
       final TreeMap<String,String> systemConfig = new TreeMap<>();
-      systemConfig
-          .putAll(shellState.getAccumuloClient().instanceOperations().getSystemConfiguration());
+      final TreeMap<String,String> siteConfig = new TreeMap<>();
+      boolean canReadSystemSiteConfig = false;
+
+      try {
+        // The user may not have permissions for system/site configs so handle security exception by
+        // wrapping
+        // in a try/catch block and setting a flag
+        systemConfig
+            .putAll(shellState.getAccumuloClient().instanceOperations().getSystemConfiguration());
+        siteConfig
+            .putAll(shellState.getAccumuloClient().instanceOperations().getSiteConfiguration());
+        canReadSystemSiteConfig = true;
+      } catch (AccumuloException | AccumuloSecurityException e) {
+        if (isSecurityException(e) && (tableName != null || namespace != null)) {
+          Shell.log.debug(
+              "User does not have permission to read system and/or site configuration: {}",
+              e.getMessage());
```

Comment:
I would put this as a warning rather than debug. This could also be separate messages, with information about what shell command to use to grant the appropriate permission.